src/server/handle-api.js

```diff
@@ -15,26 +15,35 @@ export default ({ server, config }) => {
     path: '/api/v1/{query*}',
     handler: (req, reply) => {
 
-      const base = `${config.siteUrl}/wp-json/wp/v2`
+      const base = `${normalizePath(config.siteUrl)}/wp-json/wp/v2`
       const path = `${req.params.query}${req.url.search}`
       const remote = `${base}/${path}`
+      const cacheKey = normalizePath(path)
```

Comment:
I wonder if `normalizePath` is the right name to use here? Am i correct in thinking it just strips leading and trailing slashes? Wondering if it'd help to make name tie more in to that?